microsoft/fluentui-android
# Fix BottomSheet Bug for setting Hidden State add Java APIs for (FURN android) .

### Platforms Impacted

### Description of changes

1. Add Fix for State.Hidden for BottomSheet.
2. Add JVMOverload for SheetItem Constructors/
3. Add a counter function for adding custom-view to bottomSheet required for FURN android.
4. Add a Scroll event hide/unhide sample for apps. 
5. Add content description API for drawer handle


### Verification
1. Open Persistent BottomSheet demo.
2. scroll down the view and bottomsheet should disappear.
3. scroll up and it should reappear.
4. All earlier functions should work as is.


### Video
![BottomSheet Changes](https://user-images.githubusercontent.com/60392150/111194074-d3f2b800-85e0-11eb-928b-65a6b8719f01.gif)


### Pull request [checklist]

This PR has considered:
- [ NA ] Light and Dark appearances
- [ NA ] VoiceOver and Keyboard Accessibility
- [ NA ] Internationalization and Right to Left layouts
- [ NA] Size classes and window sizes (notched devices, multitasking, different window sizes, etc)
